packages/interface/complementary-area-context/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { withPluginContext } from '@wordpress/plugins';
+
+export default withPluginContext( ( context, ownProps ) => {
```

Comment:
@afercia @aristath Just wondering if this file should be in the `src` folder of the package?

I'm not familiar with the PR, but just spotted this file when browsing the codebase, so I may be wrong.